unlock-app/src/hooks/useEns.js

```diff
@@ -1,38 +1,52 @@
 import { ethers } from 'ethers'
 import { useState, useEffect, useContext } from 'react'
 import { AuthenticationContext } from '../contexts/AuthenticationContext'
+import configure from '../config'
+
+const config = configure()
+
+export const getNameOrAddressForAddress = async (address) => {
```

Comment:
Type the address please? It makes it easy to use these functions elsewhere when you know what input and output they expect.